src/fetch-groups.js

```diff
@@ -195,9 +202,29 @@ export default async function (specs, options) {
     spec.organization = spec.organization ?? "W3C";
 
     if (!spec.groups) {
-      let groups = null;
+      // Get group info from validate-repos if possible to avoid having to
+      // send individual network requests for each spec
+      // Note: this will not yield anything for many /TR specs because we
+      // guess the name of the repo from the shortname.
```

Comment:
would https://github.com/w3c/spec-dashboard/blob/gh-pages/repo-map.json help?

Reply:
Oh yes, I had forgotten about this one (which find-specs also makes use of).

Reply:
Tracked in #1832